torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -96,6 +117,19 @@ def test_user_rrefs_confirmed_remote(self):
         )
         self.assertEqual(ret_rref.to_here(), True)
 
+    @dist_init
+    def test_global_var_rref(self):
+        dist.barrier()
+        dst = worker_name((self.rank + 1) % self.world_size)
+        rref = rpc.remote(dst, get_dict)
+        rpc.rpc_sync(dst, write_dict, args=(rref, 1, 10))
```

Comment:
The error occurs here.

Reply:
After discussing with @xush6528 and @wanchaol, it seems it does not yet work when using `RRef[Dict[...]]` or `Future[Dict[...]]` as argument types in `TorchScript` functions.